bot_cli/notes_classes.py

```diff
@@ -109,7 +109,7 @@ def show_notes(self):
         for index, data in enumerate(self.data["notes"]):
             note = str(data["note"])
             tags = [str(tag) for tag in data["tags"]]
-            notes.append({index: {"Note": note.capitalize(), "Tags": tags}})
+            notes.append({index+1: {"Note": note.capitalize(), "Tags": tags}})
```

Comment:
Thanks! I've discussed it with Yuri, but we couldn't reach an agreement that it needed to be fixed... ) Wanted to fix it myself, but didn't get to it